tests/Microsoft.Bot.Builder.Azure.Tests/AzureQueueTests.cs

```diff
@@ -40,13 +41,15 @@ public async Task ContinueConversationLaterTests()
                        .UseStorage(new MemoryStorage())
                        .UseBotState(new ConversationState(new MemoryStorage()), new UserState(new MemoryStorage()));
 
+                var queueStorage = new AzureQueueStorage(ConnectionString, queueName);
                 var dm = new DialogManager(new ContinueConversationLater()
                 {
-                    ConnectionString = ConnectionString,
-                    QueueName = queueName,
                     Date = "=addSeconds(utcNow(), 2)",
                     Value = "foo"
                 });
+
+                dm.InitialTurnState.Set<QueueStorage>(queueStorage);
```

Comment:
Are there any samples which did this in the old way?  Do we need a sample?

Reply:
There are no samples to update with this change. We do have a plan to add this to a sample later.

---
In reply to: [501233216](https://github.com/microsoft/botbuilder-dotnet/pull/4692#discussion_r501233216) [](ancestors = 501233216)